docs/architecture/grpc-for-wcf-developers/docker.md

```diff
@@ -17,8 +17,8 @@ Microsoft provides a range of base images for building and running .NET Core app
 
 | Image | Description |
 | ----- | ----------- |
-| [mcr.microsoft.com/dotnet/core/sdk](https://hub.docker.com/_/microsoft-dotnet-core-sdk/) | For building applications with `docker build`. Not to be used in production. |
-| [mcr.microsoft.com/dotnet/core/aspnet](https://hub.docker.com/_/microsoft-dotnet-core-aspnet/) | Contains the runtime and ASP.NET Core dependencies. For production. |
+| [mcr.microsoft.com/dotnet/sdk](https://hub.docker.com/_/microsoft-dotnet-core-sdk/) | For building applications with `docker build`. Not to be used in production. |
```

Comment:
All references to the .NET repos in Docker Hub should also get updated so that users won't get redirected.
```suggestion
| [mcr.microsoft.com/dotnet/sdk](https://hub.docker.com/_/microsoft-dotnet-sdk/) | For building applications with `docker build`. Not to be used in production. |
```

Reply:
Replaced all occurrences of this URL pattern.